.travis.yml

```diff
@@ -99,13 +99,25 @@ deploy:
       jdk: oraclejdk8
   - provider: script
     skip_cleanup: true
-    script: ./gradlew uploadArchives -PossrhUsername="$SONATYPE_USERNAME" -PossrhPassword="$SONATYPE_PASSWORD"
+    # wait until uploading task finishes, because it may cost more than 10 minutes without any output
+    # https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
```

Comment:
Page anchor should start with lowercase 'b'

Reply:
That page actually uses upper case as `id`, you can have a try:

* https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

Reply:
Well, the actual [standard](https://www.w3.org/TR/WD-html40-970708/struct/links.html) says that anchors are supposed to be case-insensitive, so I'll approve for merge.